src/SFA.DAS.EmployerFinance.Web/Controllers/HomeController.cs

```diff
@@ -1,14 +1,25 @@
-﻿using System.Diagnostics;
+﻿using System;
+using System.Diagnostics;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
 using SFA.DAS.EmployerFinance.Web.Models;
 
 namespace SFA.DAS.EmployerFinance.Web.Controllers
 {
     [Route("")]
     public class HomeController : Controller
     {
+        private readonly ILogger<HomeController> _logger;
+
+        public HomeController(ILogger<HomeController> logger)
```

Comment:
Could a non generic `ILogger` be used instead by leveraging StructureMap?

Reply:
I check on both the NLog Nuget page and Microsoft .net asp core pages and I could only find this approach being used. We might be able to create the injection needed for just using an ILogger but it looks like that would have to be custom code.

Reply:
done